content/rs/administering/access-control/user-roles.md

```diff
@@ -5,83 +5,73 @@ weight: $weight
 
-## Cluster Management Roles
+RBAC lets you set permissions for your databases and for the Redis Enterprise management console itself, providing a complete security-management solution for your cluster. 
 
-Each user role is assigned a management role that defines the access the user with that role has in the RS web UI and API for the cluster.
+### What is RBAC good for?
 
-The management roles are:
+Role-Based Access Control allows you to scale your Redis deployments while minimizing the overhead involved in managing a cluster with many databases, multiple users, and various access control lists. With RBAC, you can create a role once and then deploy it across multiple databases in the cluster with ease.
 
-{{< embed-html "account-role-table.html" >}}
+### Configuring Roles & Users
 
-### Assigning Management Roles to a User Role
+In **access control > roles**, you can configure various types of user roles. For example:
 
-To assign a management role to a user role:
+- Management roles that define user access to the Admin UI and cluster API
+- Data access controls with Redis ACLs that define the commands and keys that users can access when connecting to databases
+- The databases that you would like each user to be able to access
 
-1. In **access control** > **roles**:
-    - Edit an existing role - Hover over a role and click ![Edit](/images/rc/icon_edit.png#no-click "Edit").
-    - Create a new role - Click ![Add](/images/rs/icon_add.png#no-click "Add").
-1. Select the management role for the user role.
-1. Click **Save**.
+### Management Roles
 
-### User Roles for Database Connections Only
+Redis Enterprise comes with the following roles for the admin console:
 
-To create a user role for users that cannot connect to the RS web UI and API, assign the **None** management role to the user role.
+{{< embed-html "account-role-table.html" >}}
 
-## Database Access Control
+{{% note %}}
+To create a user role that only has access to the database, assign the user the management role **None**
+{{% /note %}}
 
-To control user access to Redis database commands and keys,
-you must define Redis ACLs that specify the commands that users can run and keys that the commands can apply to.
-Then, in the user role, select the databases that the users can access and Redis ACL that controls user access to those databases.
+### Database Access Controls
 
-A-A cannot assign ACLs on creation
+ACLs allow you to control what level of access each user has in Redis. With ACLs, you can specify which commands specific users can execute and which keys they can access. This allows for much better security practices: you can restrict any given user’s access to the least level of privilege needed.
```

Comment:
@IAmATeaPot418 This seems to imply that you set access on a per-user basis. I think this needs some rewording.

Reply:
How about:
**Redis** ACLs allow you to control what level of access each **role and** user have in Redis. With **Redis** ACLs, you can specify which commands specific **roles and** users can execute and which keys they can access. This allows for much better security practices: you can restrict any given user’s access to the least level of privilege needed.

Reply:
I like Alon's wording. Will add a commit with that if you agree Kyle.

Reply:
Fine by me @IAmATeaPot418

Reply:
I pushed the new wording in the last commit